cmd/quickstart.go

```diff
@@ -42,6 +45,8 @@ func runQuickStart(
 		defer f.Close()
 
 		_, err = tea.NewProgram(quickstart.NewContainerModel(
+			viper.GetBool(cliflags.AnalyticsOptOut),
```

Comment:
Seems like this should be with the other flag values at the end of the param list.

Reply:
Fair. I was putting it on top of analyticsTracker for alphabetical reasons